gen2-efficentnet-classification/main.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import argparse
+from pathlib import Path
+import sys
+import cv2
+import depthai as dai
+import numpy as np
+from classes import class_names
+
+# Get Argument First
+parser = argparse.ArgumentParser()
+parser.add_argument('-nd', '--no-debug', action="store_true", help="Prevent debug output")
+parser.add_argument('-cam', '--camera', action="store_true", help="Use DepthAI 4K RGB camera for inference (conflicts with -vid)")
+parser.add_argument('-vid', '--video', type=str, help="Path to video file to be used for inference (conflicts with -cam)")
+args = parser.parse_args()
+
+
+
+# NOTE: video must be of size 224 x 224. We will resize this on the
+# host, but you could also use ImageManip node to do it on device
+
+# Link video in with the detection network
+    
+if not args.camera and not args.video:
+    raise RuntimeError("No source selected. Please use either \"-cam\" to use RGB camera as a source or \"-vid <path>\" to run on video")
+
+debug = not args.no_debug
+camera = not args.video
+class_names = class_names()
```

Comment:
```suggestion
labels = class_names()
```
This line shadowed the imported `class_names` function